adoc/chapters/what_changed.adoc

```diff
@@ -4,6 +4,8 @@
 [[cha:what-changed-from]]
 = What has changed from previous versions
 
+The device query [code]#info::fp_config::correctly_rounded_divide_sqrt# has
+been deprecated.
```

Comment:
I assume it is our intention that this change is a "fix" to SYCL 2020, right?  Or, is it our intention to make this change in SYCL-Next?

If it is a fix to SYCL 2020, then this paragraph should go in the section "What has changed from SYCL 1.2.1 to SYCL 2020".  In that case, move this paragraph to the end of that section.

Reply:
Oh I see, that makes sense, I moved the paragraph in the SYCL 2020 section next to other device aspects related changes.